app/models/question_option.rb

```diff
@@ -52,6 +52,10 @@ class QuestionOption < ActiveRecord::Base
   scope :by_number, -> { order(:number) }
 
 
+  # ===========================
+  # = Public instance methods =
+  # ===========================
+
```

Comment:
Probably just need the one comment block ;)
No need to hold off on merging due to this though